server/src/db/seeders/PastaSeeder.js

```diff
@@ -0,0 +1,41 @@
+import { Pasta } from "../../models/index.js";
+
+class PastaSeeder {
+  static async seed() {
+    const pastasData = [
+      {
+        name: "spaghetti",
+        description: "Long Noodle good for chicken parm!",
+        categoryId: 2,
```

Comment:
This is fine to hard code `id`'s but we should ideally be querying for the exact category that we want to associate

Reply:
can use - await Category.query().findOne({name: "long"})